src/main/kotlin/no/fdk/dataset_catalog/extensions/DatasetExtensions.kt

```diff
@@ -1,5 +1,258 @@
 package no.fdk.dataset_catalog.extensions
 
+import no.fdk.dataset_catalog.configuration.ApplicationProperties
 import no.fdk.dataset_catalog.model.*
+import java.time.LocalDateTime
 
-fun List<Dataset>.toDTO() : DatasetEmbeddedWrapperDTO = DatasetEmbeddedWrapperDTO(mapOf(Pair("datasets", this)))
+fun List<Dataset>.toDTO(): DatasetEmbeddedWrapperDTO = DatasetEmbeddedWrapperDTO(mapOf(Pair("datasets", this)))
+
+fun Dataset.datasetToDBO(): DatasetDBO =
+    DatasetDBO(
+        id = id!!,
+        catalogId = catalogId!!,
+        uri = uri,
+
+        lastModified = lastModified,
+        specializedType = specializedType,
+        originalUri = originalUri,
+        published = registrationStatus == REGISTRATION_STATUS.PUBLISH,
+        approved = registrationStatus == REGISTRATION_STATUS.APPROVE || registrationStatus == REGISTRATION_STATUS.PUBLISH,
+        concepts = concepts?.mapNotNull { it.uri }?.toSet(),
+        title = title?.let { LocalizedStrings(it["nb"], it["nn"], it["en"]) },
```

Comment:
Vil ikke dette feile om `title`mangler "nb", "nn" eller "en"?

Reply:
Bruk heller get-funksjonen. Den svarer med verdien `null` når den ikke finnes, i stedet for å feile
```suggestion
        title = title?.let { LocalizedStrings(it.get("nb"), it.get("nn"), it.get("en")) },
```